### PR TITLE
432 version 1.2

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

File: R/utilities-plots.R [hunk 1]
```diff
@@ -1,20 +1,7 @@
-# Currently, need to load a RData because, jsonlite need to be in Namespace/Description otherwise
-# The code for loading the json is consequently left commented
-# reThemeProperties <- jsonlite::fromJSON('./data/RE-theme.json')
-load("./data/reThemeProperties.RData")
```
Comment (Yuri05) [inline R/utilities-plots.R:4]: yesss ! 👍

**Review by Yuri05 — APPROVED** [review]

**Comment by Yuri05** [comment]
something wrong with vignettes now...
```
--- re-building 'goodness-of-fit.Rmd' using rmarkdown
[WARNING] This document format requires a nonempty <title> element.
  Defaulting to 'Report.utf8' as the title.
  To specify a title, use 'title' in metadata or --metadata title="...".
Quitting from lines 162-205 (goodness-of-fit.Rmd) 
Error: processing vignette 'goodness-of-fit.Rmd' failed with diagnostics:
'gpar' element 'col' must not be length 0
--- failed re-building 'goodness-of-fit.Rmd'
```

**Comment by pchelle** [comment]
> 
> 
> something wrong with vignettes now...
> 
> ```
> --- re-building 'goodness-of-fit.Rmd' using rmarkdown
> [WARNING] This document format requires a nonempty <title> element.
>   Defaulting to 'Report.utf8' as the title.
>   To specify a title, use 'title' in metadata or --metadata title="...".
> Quitting from lines 162-205 (goodness-of-fit.Rmd) 
> Error: processing vignette 'goodness-of-fit.Rmd' failed with diagnostics:
> 'gpar' element 'col' must not be length 0
> --- failed re-building 'goodness-of-fit.Rmd'
> ```

Hmmm. I tried to reproduce but on my end I get the following error when using 'ospsuite' dev version no matter how I load the package:
Error: object 'getCurrentConvertedObject' not found
   Error: .onLoad failed in loadNamespace() for 'ospsuite', details:
     call: rClr::clrCallStatic("OSPSuite.R.Api", paste0("Get", taskName))
     error: Type:    RDotNet.EvaluationException
   Message: Error: object 'getCurrentConvertedObject' not found

**Comment by pchelle** [comment]
I have investigated the error message.
It seems the issue comes from the function 'gpar' from the package 'grid' on which 'ggplot2' is built.
Since, there does not seem to be any issue on my computer, I am thinking this problem might be due to the version of the 'grid' package.
Do we have a way of checking the version when appveyor build the package ?

**Comment by Yuri05** [comment]
> Do we have a way of checking the version when appveyor build the package ?

Hmm, I have no idea. @msevestre Do you know? @pchelle which grid-package version are you using locally?

**Comment by msevestre** [comment]
nope no idea. If you need a specific version, why don't you depent on it somehow or install it explicitly so that you don;t have to worry about it?

**Comment by pchelle** [comment]
My local grid version is 3.6.0
However, not sure if the problem is grid itself as ggplot2 depends on grid, so a different ggplot2 version could also cause it.
I am going to try Michael's suggestion, there may be some failures in the process though.

**Comment by Yuri05** [comment]
@pchelle Looks good. Ready to merge?

**Comment by pchelle** [comment]
> 
> 
> @pchelle Looks good. Ready to merge?

Yes !

**Comment by msevestre** [comment]
LEt's merge then

**Pull request merged**